### ISSUE TITLE
GetMemoryInfo not implemented Error for get_memory_info

### ISSUE THREAD

**Issue by mfatih7** [pytorch/xla#5968]
Hello

I was using 

`print( 'Device is ' + xm.xla_real_devices([str(device)])[0] + ' with ' + str(xm.get_memory_info(device)['kb_free']) + ' KB free memory ' + str(xm.get_memory_info(device)['kb_total']) + ' KB total memory '  )    `

command to observe the TPU memory utilization.
I think after switching from XRT to PJRT I cannot use the command anymore and I get the error below.

```
line 18, in <module>
    print( 'Device is ' + xm.xla_real_devices([str(device)])[0] + ' with ' + str(xm.get_memory_info(device)['kb_free']) +
' KB free memory ' + str(xm.get_memory_info(device)['kb_total']) + ' KB total memory '  )
RuntimeError: ./torch_xla/csrc/runtime/pjrt_computation_client.h:137 : GetMemoryInfo not implemented
```
How can I observe the TPU memory utilization now?

**Comment by JackCaoG** [comment]
right.. this is a known problem, we removed this in the new PJRT runtime since there is no direct replacement. We have been using profiler to observe the memory trend. @will-cromar if you have ant idea if it is possible to add similar feature to PJRT

**Comment by mfatih7** [comment]
What about observing the memory utilization of each TPU core individually when model-level parallelization of SPMD is used?
With the profiler, I can only observe the memory utilization of TPU core 0 for a TPUv3-8 device.

**Comment by JackCaoG** [comment]
I can turn this into a RFC but I am not sure if we can have anyone to implement this soon.
